Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN pip3 install pipenv
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-WORKDIR /usr/src/app
+WORKDIR /usr/src/udk
```

Comment:
We should update the client/Dockerfile as well.

Also this code could be ripped out into its own PR.

Reply:
Move to its own PR https://github.com/elthran/UDK/pull/23